Source/Application/Constants/UserStorage.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UserStorage.swift
+//  GithubNotifications
+//
+//  Created by Chananchida F. on 9/15/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import Foundation
+
+enum UserStorage {
```

Comment:
You will use this in a Singleton way, I suggest creating a protocol that will help with unit test. Also I suggest making this an object with `get` and `set` functions.